docs/src/pages/[platform]/connected-components/liveness/faqs-android.mdx

```diff
@@ -0,0 +1,3 @@
+### (Android-specific) When creating a backend using a REST API, seeing compilation errors relating to `com.amazonaws.mobileconnectors.apigateway.annotation`
+
+As per our [documentation](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android/#configure-api), these autogenerated files can be safely deleted — this should be a one-time fix after `amplify add api` and `amplify push`.
```

Comment:
@tjleing We should bias towards descriptive link text that helps with SEO link juice. Can you update this link to read "Configure Amplify API Android documentation"